exercises/concept/weather-forecast/weather_forecast_test.go

```diff
@@ -112,7 +112,7 @@ func testComment(entityKind, entityName, comment, wantedPrefix string) (ok bool,
 	lowerEntity := strings.ToLower(entityKind)
 
 	// Check if comment has wanted prefix
-	if !strings.HasPrefix(comment, wantedPrefix) {
+	if !strings.HasPrefix(trimmedComment, wantedPrefix) {
```

Comment:
The `testComment` function you modified here is also used for other entities than the package level comment. However, to use block comments e.g. for a function comment would not be idiomatic to Go.

I think the best way to handle this would be to create a separate version of this function and use that when checking the package level comment.

Reply:
I'm not sure I see your point. The tests pass block comments and line comments identically, for package comments and function comments. If you think that that behaviour is incorrect, I suggest that it's a separate issue altogether. The problem that this fix addresses is that the test author trimmed the spaces, which are not consequential to the code comment, but then tested against the raw comment string instead of the trimmed comment.

Reply:
Possibly relevant to see how godoc itself parses code comments (see TestSanitizeFunc for examples of it stripping new lines and tabs, basically it uses HTMLEncode, which strip whitespace.)
https://go.googlesource.com/tools/+/refs/heads/master/godoc/godoc_test.go

Reply:
Sorry for the confusion. I did not properly check what exactly is used as input for the function. You are right, the PR is ok as is and recommending line comments would be another issue entirely.